gr-qtgui/examples/show_controls.grc

```diff
@@ -0,0 +1,315 @@
+    comment: ''
+    fontColor: default
+    gui_hint: 2,4,1,1
+    isVertical: 'False'
+    label: Gain
+    maximum: '100'
+    minimum: '0'
+    msize: '300'
+    position: '2'
+    scaleFactor: '1'
+    showValue: 'True'
+    type: int
+    value: gain
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [928, 340.0]
+    rotation: 0
+    state: enabled
+- name: ledstatus
+  id: variable_qtgui_toggle_button_msg
+  parameters:
+    comment: ''
+    gui_hint: 0,2,1,1
+    initPressed: 'True'
+    label: LED
+    pressBackgroundColor: green
+    pressFontColor: default
+    pressed: 'True'
+    relBackgroundColor: red
+    relFontColor: default
+    released: 'False'
+    type: bool
+    value: 'True'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [552, 12.0]
+    rotation: 0
+    state: true
+- name: samp_rate
+  id: variable
+  parameters:
+    comment: ''
+    value: '32000'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [184, 12]
+    rotation: 0
+    state: enabled
+- name: blocks_null_sink_0
+  id: blocks_null_sink
+  parameters:
+    affinity: ''
+    alias: ''
+    bus_structure_sink: '[[0,],]'
+    comment: ''
+    num_inputs: '1'
+    type: complex
+    vlen: '1'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [696, 448.0]
+    rotation: 0
+    state: true
+- name: blocks_null_source_0
+  id: blocks_null_source
+  parameters:
+    affinity: ''
+    alias: ''
+    bus_structure_source: '[[0,],]'
+    comment: ''
+    maxoutbuf: '0'
+    minoutbuf: '0'
+    num_outputs: '1'
+    type: complex
+    vlen: '1'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [312, 448.0]
+    rotation: 0
+    state: true
+- name: blocks_throttle_0
+  id: blocks_throttle
+  parameters:
+    affinity: ''
+    alias: ''
+    comment: ''
+    ignoretag: 'True'
+    maxoutbuf: '0'
+    minoutbuf: '0'
+    samples_per_second: samp_rate
+    type: complex
+    vlen: '1'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [520, 444.0]
+    rotation: 0
+    state: true
+- name: qtgui_compass_0
+  id: qtgui_compass
+  parameters:
+    affinity: ''
+    alias: ''
+    backgroundColor: white
+    comment: ''
+    fullNeedle: '1'
+    gui_hint: 1,1,2,2
+    min_size: '150'
+    name: Azimuth
+    needleBodyColor: black
+    needleTipColor: red
+    position: '1'
+    scaleColor: black
+    setDebug: 'False'
+    update_time: '0.10'
+    usemsg: 'True'
+  states:
+    bus_sink: false
+    bus_source: false
+    bus_structure: null
+    coordinate: [336, 244.0]
+    rotation: 0
+    state: true
+- name: qtgui_graphicitem_0
+  id: qtgui_graphicitem
+  parameters:
+    affinity: ''
+    alias: ''
+    comment: ''
+    file: /opt/sdr/gnuradio/Flowgraphs/Production/noaa19.jpeg
```

Comment:
Can we put this or another example image in the `apps/` or `examples/` folder and then use a subprocess hack like here https://github.com/gnuradio/gnuradio/pull/3105 to generate the path for the installed file?